include/boost/geometry/algorithms/detail/sections/sectionalize.hpp

```diff
@@ -628,10 +629,9 @@ struct sectionalize_polygon
                 ring_identifier ring_id,
                 std::size_t max_count)
     {
-        typedef typename point_type<Polygon>::type point_type;
         typedef sectionalize_range
             <
-                closure<Polygon>::value, Reverse, point_type, DimensionVector
+                closure<Polygon>::value, Reverse, Point, DimensionVector
```

Comment:
So here instead of passing `Point` into `secionalize_range` you could remove this template parameter and instead inside `secionalize_range::apply()` take it from `Sections` as:
```
typename geometry::point_type<typename Sections::box_type>::type
```

Reply:
:heavy_check_mark: done

I moved it to in `sectionalize_part` which makes it now simpler than it was before.

(though the diff might be now larger because I changed some `typedef` with `using`)

Thanks for your comment.